python/mscclpp_benchmark/mscclpp_op.py

```diff
@@ -418,3 +418,82 @@ def auto_tune(self):
             for block_size in block_size_to_try:
                 self.set_params(nblocks, block_size)
                 yield nblocks, block_size
+
+
+class MscclppAllReduce6:
```

Comment:
@aashaka this is where one can setup a nvls connection and use it for allreduce.